### PR TITLE
🔥 On-demand test

### PR DESCRIPTION
**Pull request by thibaultleouay**
## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation

### Before this PR

{Please add a screenshot here}

### After this PR

{Please add a screenshot here}

### CODE REVIEW

File: apps/server/src/v1/monitors/results/get.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+              latency: z.number().int(), // in ms
+              statusCode: z.number().int().nullable().default(null),
+              monitorId: z.string().default(""),
+              url: z.string().url().optional(),
+              error: z
+                .number()
+                .default(0)
+                .transform((val) => val !== 0),
+              region: z.enum(flyRegions),
+              cronTimestamp: z.number().int().optional(),
```
Comment (mxkaske) [inline apps/server/src/v1/monitors/results/get.ts]: shouldn't we return the value `timestamp` instead, as it has nothing to do with `cron`.

File: packages/tinybird/src/os-client.ts [hunk 2]
```diff
@@ -460,6 +460,53 @@ export class OSTinybird {
...
+            latency: z.number().int(), // in ms
+            statusCode: z.number().int().nullable().default(null),
+            monitorId: z.string().default(""),
+            url: z.string().url().optional(),
+            error: z
+              .number()
+              .default(0)
+              .transform((val) => val !== 0),
+            region: z.enum(flyRegions),
+            cronTimestamp: z.number().int().optional(),
```
Comment (mxkaske) [inline packages/tinybird/src/os-client.ts]: `timestamp`

File: apps/server/src/v1/monitors/trigger/post.ts [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+          eq(monitor.workspaceId, Number(workspaceId)),
+          isNull(monitor.deletedAt),
+        ),
+      )
+      .get();
+
+    if (!monitorData) {
+      throw new HTTPException(404, { message: "Not Found" });
+    }
+
```
Comment (mxkaske) [inline apps/server/src/v1/monitors/trigger/post.ts:83]: check if user has exceeded the workspace `limit`

File: packages/tinybird/src/os-client.ts (hunk 2 above)
Comment (thibaultleouay) [reply packages/tinybird/src/os-client.ts]: but we are returning the timestamp is the time at with the request was actually triggered 
crontimestamp is the time when we asked for the request to be trigger
IMO we should return the time we asked for the job to run

File: apps/server/src/v1/monitors/results/get.ts (hunk 1 above)
Comment (thibaultleouay) [reply apps/server/src/v1/monitors/results/get.ts]: but the cronTimestamp is the 'scheduled timestamp' 
it's the time we have trigger the job

Comment (mxkaske) [reply apps/server/src/v1/monitors/results/get.ts]: OK but it shouldn't be `cronTimestamp` as its confusing for the user OR we rename the key. 

Suggestion: let's use `transform` to map cronTimestamp value to timestamp value.

File: apps/server/src/v1/monitors/trigger/post.ts [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+    const timestamp = Date.now();
+
+    const newRun = await db
+      .insert(monitorRun)
+      .values({
+        monitorId: row.id,
+        workspaceId: row.workspaceId,
+        runnedAt: new Date(timestamp),
+      })
+      .returning();
```
Comment (mxkaske) [inline apps/server/src/v1/monitors/trigger/post.ts:109]: per month?

File: apps/server/src/v1/monitors/trigger/post.ts [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+    const { id } = c.req.valid("param");
+    const limits = c.get("limits");
+
+    const count = (
+      await db
+        .select({ count: sql<number>`count(*)` })
+        .from(monitorRun)
+        .where(and(eq(monitorRun.workspaceId, Number(workspaceId))))
+        .all()
+    )[0].count;
```
Comment (mxkaske) [inline apps/server/src/v1/monitors/trigger/post.ts]: per month?

And we might wanna add it to the `/v1/checker` api endpoint

Comment (thibaultleouay) [reply apps/server/src/v1/monitors/trigger/post.ts]: let's remove api/checker 🤣

File: apps/server/src/v1/monitors/results/get.ts (hunk 1 above)
Comment (thibaultleouay) [reply apps/server/src/v1/monitors/results/get.ts]: but let's return the actual time of the request the timestamp make more sense you are right

File: apps/web/src/config/pricing-table.ts [hunk 6]
```diff
@@ -1,10 +1,15 @@
-import type { LimitsV2 } from "@openstatus/db/src/schema/plan/schema";
+import type { LimitsV1, LimitsV2 } from "@openstatus/db/src/schema/plan/schema";
 
 export const pricingTableConfig: Record<
   string,
   {
     label: string;
-    features: { value: keyof LimitsV2; label: string; badge?: string }[];
+    features: {
+      value: keyof LimitsV1 | keyof LimitsV2;
```
Comment (mxkaske) [inline apps/web/src/config/pricing-table.ts:8]: FIXME:

**Pull request merged**